dotCMS/src/main/java/com/dotmarketing/business/IdentifierFactoryImpl.java

```diff
@@ -545,8 +547,6 @@ protected String getAssetTypeFromDB(String identifier) throws DotDataException{
 					.format("Error trying find the Asset Type from identifier=[%s]: %s", identifier, e.getMessage()));
 			throw new DotDataException(String
 					.format("Error trying find the Asset Type from identifier=[%s]", identifier), e);
-		} finally{
-			DbConnectionFactory.closeConnection();
```

Comment:
good

Reply:
@acurionedotcms did we miss the annotation on the method here? maybe was incorrectly removed after as part of the last changes

Reply:
yeah same question

Reply:
Daniel suggested to leave it like that.. he commented by mistake but later said this is fine.